spec/controllers/admin/action_call_dashboard_controller_spec.rb

```diff
@@ -15,20 +16,23 @@
   end
 
   describe "GET #index" do
-    it "assigns all admin_action_call_infos as @admin_action_call_infos ordered by call_count descending" do
+    it "renders the inertia page with admin_action_call_infos ordered by call_count descending" do
       admin_action_call_info1 = create(:admin_action_call_info, call_count: 3)
       admin_action_call_info2 = create(:admin_action_call_info, action_name: "stats", call_count: 5)
 
       get :index
 
-      expect(assigns(:admin_action_call_infos)).to eq([admin_action_call_info2, admin_action_call_info1])
-    end
-
-    it "renders the index template" do
-      get :index
-
       expect(response).to have_http_status(:ok)
-      expect(response).to render_template(:index)
+      expect_inertia.to render_component "Admin/ActionCallDashboard/Index"
+      expect(inertia.props[:admin_action_call_infos]).to eq([admin_action_call_info2, admin_action_call_info1].map do |info|
+        {
+          id: info.id,
+          controller_name: info.controller_name,
+          action_name: info.action_name,
+          call_count: info.call_count
+        }
+      end
+      )
```

Comment:
```suggestion
      end)
```

(or something, not sure what the best formatting is but I don't think it's this)